### PR TITLE
refactor: remove init from cobra command loading

### PR DESCRIPTION
**Pull request by ravisuhag**

### CODE REVIEW

File: cmd/serve.go [hunk 1]
```diff
@@ -5,14 +5,15 @@ import (
...
-	})
-}
-
-func serve(cmd *cobra.Command, args []string) {
-	server.Start()
+// ServeCmd start new stencil server
+func ServeCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:     "serve",
+		Aliases: []string{"v"},
```
Comment (kushsharma) [inline cmd/serve.go:12]: what does `v` stands for?

File: cmd/upload.go [hunk 2]
```diff
@@ -10,51 +10,53 @@ import (
...
-		Version:   version,
-		Latest:    latest,
-		Data:      fileData,
-	}
-	res, err := client.UploadDescriptor(context.Background(), ur)
-	if err != nil {
-		log.Fatalln(err)
-	}
-	log.Println(res)
+	cmd.Flags().StringVarP(&host, "host", "h", "", "stencil host address eg: localhost:8000")
```
Comment (harikrishnakanchi) [inline cmd/upload.go:50]: This seems to be failing with following errors

```
panic: unable to redefine 'h' shorthand in "upload" flagset: it's already used for "host" flag
```

File: cmd/serve.go (hunk 1 above)
Comment (ravisuhag) [reply cmd/serve.go:12]: Typo. Fixing it.

File: cmd/upload.go (hunk 2 above)
Comment (ravisuhag) [reply cmd/upload.go:50]: Weird one. Global -h for help was overlapping.

**Pull request merged**